comprehensive/lms/static/cookieBanner.js

```diff
@@ -51,4 +52,20 @@ oxa = window.oxa || {};
 
 (oxa));
 
-var cookieNotice = new oxa.cookieBanner();
\ No newline at end of file
+var cookieNotice = new oxa.cookieBanner();
+
+// track the user clicks, anchor/button and exclude the LearnMore link on the banner itself
+window.onload = function () {
+    document.onclick = function (event) {
+        event = event || window.event;
+        var target = event.target || event.srcElement;
+
+        while (target) {
+            if ((target instanceof HTMLAnchorElement || target instanceof HTMLButtonElement) && target.id != 'btnPrivacy') {
```

Comment:
okay - i've thought about it some more. i think we'd be better off registering two click event listeners (on hyperlinks and buttons). the implementation function would ensure the element doesn't have "btnprivacy" id before dismissing the banner. this should result in about the same lines of code, but should be much more performant because we won't exhaust the whole DOM tree on each click event AND I think it'll be more readable. We can also remove the listeners when the cookie banner is closed.